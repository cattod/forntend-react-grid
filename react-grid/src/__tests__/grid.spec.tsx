import * as React from "react";
import {CatodGrid} from "../index";
import {create} from "react-test-renderer";

test("Component should show 'red' text 'Hello World'", () => {
  const component = create(<CatodGrid  message="400px"/>);
  const testInstance = component.root;

  expect(testInstance.findByType(CatodGrid).props.message).toBe("400px");

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});