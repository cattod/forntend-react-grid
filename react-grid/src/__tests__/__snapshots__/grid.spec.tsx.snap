// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component should show 'red' text 'Hello World' 1`] = `
<div
  className="grid-body"
>
  <table
    className="table table-bordered table-sm table-hover"
  >
    <thead
      className="back-header"
    >
      <tr />
    </thead>
    <tbody />
  </table>
  <p
    className="empty-grid"
  >
    400px
  </p>
</div>
`;
